centaurCwlRunner/src/main/scala/centaur/cwl/Outputs.scala

```diff
@@ -0,0 +1,106 @@
+package centaur.cwl
+
+import cats.effect.IO
+import centaur.api.CentaurCromwellClient
+import cromwell.api.model.SubmittedWorkflow
+import cwl.{CwlDecoder, CwlType, MyriadOutputType, File => CwlFile}
+import io.circe.Json
+import io.circe.generic.auto._
+import io.circe.literal._
+import io.circe.refined._
+import io.circe.shapes._
+import io.circe.syntax._
+import shapeless.{Inl, Poly1}
+import spray.json.{JsNumber, JsObject, JsString, JsValue}
+
+import scalaz.syntax.std.map._
+
+object Outputs {
+
+  //When the string returned is not valid JSON, it is effectively an exception as CWL runner expects JSON to be returned
+  def handleOutput(submittedWorkflow: SubmittedWorkflow): String = {
+    val metadata: Map[String, JsValue] = CentaurCromwellClient.metadata(submittedWorkflow).get.value
+
+    //Sorry for all the nesting, but spray json JsValue doesn't have optional access methods like Argonaut/circe,
+    //thus no for comprehensions for us :(
+    metadata.get("submittedFiles.workflow") match {
+      case Some(JsString(some)) =>
```

Comment:
ToL: `some` to `stringValue`? Or even better what the value represents. IMO the `Some`ness is implied